plugins/aws-s3-storage/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM verdaccio/verdaccio:4
```

Comment:
Great, stuff, could we update the readme so people is aware of this?

Reply:
done